src/deepsparse/transformers/pipelines/text_generation.py

```diff
@@ -282,6 +294,8 @@ def process_inputs(self, inputs: TextGenerationInput) -> List[numpy.ndarray]:
             self.engine.session_id = inputs.session_id
             self.multitoken_engine.session_id = inputs.session_id
 
+        self.streamer = inputs.streamer
```

Comment:
why are we assigning state to the model based on an input? would prefer to keep in the context of a single inference as this may cause issues with multi stream and other scenarios

Reply:
@bfineran I couldn't find a way to pass kwargs into the map function used by the threadpool (as I tried adding the streamer to `postprocess_kwargs` and providing `postprocess_kwargs` to `engine_forward()`
```python
            # submit split batches to engine threadpool
            batch_outputs = list(self.executor.map(self.engine_forward, batches))
```